examples/morphir-elm-projects/unit-test-framework/example-project-tests/src/ExampleTests/PassingModule.elm

```diff
@@ -0,0 +1,90 @@
+module ExampleTests.PassingModule exposing (..)
+
+import Example.ExampleModule exposing (..)
+import Morphir.SDK.Dict as Dict
+import Morphir.UnitTest.Expect as Expect
+import Morphir.UnitTest.Test as Test exposing (..)
+import Set
+
+
+
+--Prevents introspection from running, to ensure that the test framework doesn't break if introspection isn't available
+
+
+breakIntrospection1 : (a -> Expect.Expectation) -> a -> Expect.Expectation
+breakIntrospection1 f x =
+    f x
+
+
+breakIntrospection2 : (a -> a -> Expect.Expectation) -> a -> a -> Expect.Expectation
+breakIntrospection2 f x y =
+    f x y
+
+
+simplePassingTests : Test
+simplePassingTests =
+    describe "Suite of passing tests"
+        [ test "equal" <|
+            \_ -> Expect.equal 0 0
+        , test "notEqual" <|
+            \_ -> Expect.notEqual 0 1
+        , test "lessThan" <|
+            \_ -> Expect.lessThan 0 1
+        , test "greaterThan" <|
+            \_ -> Expect.greaterThan 2 1
+        , test "atMost" <|
+            \_ -> Expect.atMost 0 0
+        , test "atLeast" <|
+            \_ -> Expect.atLeast 2 1
+        , test "okay" <|
+            \_ -> Expect.okay <| stringToColor "Red"
+        , test "err" <|
+            \_ -> Expect.err <| stringToColor "Brazil"
+        , test "assert ==" <|
+            \_ -> Expect.assert (1 == 1)
+        , test "assert >=" <|
+            \_ -> Expect.assert (1 >= 1)
+        , test "pass" <|
+            \_ -> Expect.pass
+        , test "When cannot introspect 1" <|
+            \_ -> breakIntrospection1 Expect.err (stringToColor "Brazil")
+        , test "When cannot introspect 2" <|
+            \_ -> breakIntrospection2 Expect.equal 0 0
+        ]
+
+
+passingCollectionTests : Test
+passingCollectionTests =
+    Test.concat <|
```

Comment:
Why is it necessary to specify `Test.concat` here, but other `Test.` functions are just named directly?

Reply:
You had mentioned to me in Slack that you were intending to use Test.concat here... this needs to be Test.concat or be wrapped in a describe.